includes/parser/Parser.php

```diff
@@ -3764,13 +3771,25 @@ function braceSubstitution( $piece, $frame ) {
 	 * and its redirect destination title. Cached.
 	 *
 	 * @param $title Title
+	 * @param $args array
+	 * @param $frame PPFrame_DOM
 	 *
 	 * @return array
 	 */
-	function getTemplateDom( $title ) {
+	function getTemplateDom( $title, $args, $frame ) {
```

Comment:
Have you checked other calls to this method in the code base? We are modifying this method's signature